packages/web-client/app/components/card-space/create-space-workflow/index.ts

```diff
@@ -185,6 +244,43 @@ class CreateSpaceWorkflow extends Workflow {
       message:
         'You attempted to restore an unfinished workflow, but your Card Wallet got disconnected. Please restart the workflow.',
     }),
+    new WorkflowMessage({
+      message: `It looks like you haven't created a business account yet. In order to create your Card Space, you must first create your first business account. This is required
+          because your Card Space URL will depend on your business account ID.`,
+      includeIf() {
+        return (
+          this.workflow?.cancelationReason ===
+            FAILURE_REASONS.NO_BUSINESS_ACCOUNT ||
+          this.workflow?.cancelationReason ===
+            FAILURE_REASONS.RESTORATION_NO_BUSINESS_ACCOUNT
+        );
+      },
+    }),
+    new WorkflowMessage({
+      message: `It looks like you all your business accounts have already been used to create a Card Space. In order to create your Card Space, you must first create a new business account.`,
+      includeIf() {
+        return (
+          this.workflow?.cancelationReason ===
+            FAILURE_REASONS.ALL_BUSINESS_ACCOUNTS_TAKEN ||
+          this.workflow?.cancelationReason ===
+            FAILURE_REASONS.RESTORATION_ALL_BUSINESS_ACCOUNTS_TAKEN
```

Comment:
I think a reason why we have `RESTORATION_` counterparts of the `FAILURE_REASONS` is because of different copy. If we expect to be handling it the same way, do we need a `RESTORATION_` prefix? Could we use the `ALL_BUSINESS_ACCOUNTS_TAKEN` cancelation reason during restoration?

Reply:
I made an attempt to add the check for this during restoration, but it introduced many rendering side effects  😔 I think this needs a more thorough look so I created a [separate ticket](https://linear.app/cardstack/issue/CS-3016/support-async-restoration-checks-for-merchant-availability) for that. I think this is good to go without the merchant restoration checks since this is more of an edge case.